### PR TITLE
Revise precedence of `latest` when installing `edge`

### PR DESCRIPTION
**Pull request by crisptrutski**
Refs #753

Addresses unexpected behaviour mentioned in comment

### CODE REVIEW

File: lib/package.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ function getVersionMatch(pkgVersion, versions, options) {
 
   if (!pkgVersion) {
     var latest = options && options.latestVersion && versions[options.latestVersion];
-    if (latest)
+    if (!edge && latest)
```
Comment (guybedford) [inline lib/package.js]: :+1:

**Comment by guybedford** [comment]
All looks good. Agree the `--edge` range should be replaced by the actual. This can be done by enabling this condition to apply for edge as well - https://github.com/jspm/jspm-cli/pull/761/files#diff-f6618e1cc731d58106a806b7679a7616R225.

File: lib/install.js [hunk 2]
```diff
@@ -110,9 +110,10 @@ exports.install = function(targets, options) {
     else if (targets && options.doUpdate)
       Object.keys(targets).forEach(function(module) {
```
Comment (guybedford) [inline lib/install.js]: Can we comment what this whole block is doing?

**Comment by guybedford** [comment]
Better link to that code - https://github.com/jspm/jspm-cli/blob/master/lib/install.js#L224.

**Comment by crisptrutski** [comment]
@guybedford took a feather from cap, but discarding user supplied ranges only when necessary (since we don't provide edge-range literals to store)

File: lib/install.js [hunk 3]
```diff
@@ -227,6 +228,13 @@ function install(name, target, options, seen) {
       else
         target.setVersion(resolution.version);
     }
+    else if(options.edge && !options.parent && !options.link) {
```
Comment (guybedford) [inline lib/install.js]: space after `else if (`

**Comment by guybedford** [comment]
:+1:

**Pull request merged**